src/routes/users.routes.ts

```diff
@@ -7,8 +7,7 @@ export const useUserRoutes = (router:Express) => {
     router.post(URL_BASE + '/logout', usersController.logout);
```

Comment:
definitely need to authenticate for this in the future. Don't want to let anyone invalidate a login token